app/views/browse/deployment.html

```diff
@@ -83,7 +83,17 @@
                     <uib-tab-heading>Environment</uib-tab-heading>
                     <div ng-repeat="container in deployment.spec.template.spec.containers">
                       <h3>Container {{container.name}} Environment Variables</h3>
-                      <environment env-vars="container.env" ng-if="container.env.length"></environment>
+                      <p>
+                        <span class="pficon pficon-info" aria-hidden="true"></span>
+                        Environment variables can be edited on the <a ng-href="{{deploymentConfigName | navigateResourceURL : 'DeploymentConfig' : deployment.metadata.namespace}}?tab=environment">deployment configuration</a>.
```

Comment:
use the configURLForResource filter instead

``
{{deployment | configURLForResource}}
``